open_prices/api/urls.py

```diff
@@ -23,6 +23,7 @@
 router.register(r"v1/products", ProductViewSet, basename="products")
 router.register(r"v1/proofs", ProofViewSet, basename="proofs")
 router.register(r"v1/prices", PriceViewSet, basename="prices")
+router.register(r"v1/price_tags", PriceTagViewSet, basename="price-tags")
```

Comment:
```suggestion
router.register(r"v1/price-tags", PriceTagViewSet, basename="price-tags")
```

Reply:
recommended to use dash (hyphen) instead of underscore in URLs

Reply:
(yeah i know the gemini url doesn't match I'll take the time to refactor at somepoint :)